frontend/tests/specs/home-page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import AxeBuilder from "@axe-core/playwright";
+import { HomePage, expect, test } from "../fixtures/page-fixtures";
+
+test.describe("Home Page", () => {
+  test.beforeEach(async ({ homePage }) => {
+    await homePage.goto("/home");
+    const { topicsDropdown } = homePage.topicsDropdown;
+    await topicsDropdown.waitFor({ state: "visible" });
+  });
+
+  /* *********************************************************** */
```

Comment:
Quick note here @aasimsyed, let's use `MARK`s so it's a bit easier to navigate and we don't need to copy these large comment blocks from file to file for consistency :) I can fix this!